src/components/model/ModelStatusRollup.svelte

```diff
@@ -0,0 +1,254 @@
+    error: 'Activity type extraction has errors',
+    extracting: 'Extracting activity types',
+    none: '',
+  };
+  const parameterLogTooltipMessages: Record<ModelStatus, string> = {
+    complete: 'Model parameter extraction complete',
+    error: 'Model parameter extraction has errors',
+    extracting: 'Extracting model parameters',
+    none: '',
+  };
+  const resourceLogTooltipMessages: Record<ModelStatus, string> = {
+    complete: 'Resource type extraction complete',
+    error: 'Resource type extraction has errors',
+    extracting: 'Extracting resource types',
+    none: '',
+  };
+
+  let activityLog: ModelLog | null = null;
+  let activityLogStatus: ModelStatus = 'none';
+  let parameterLog: ModelLog | null = null;
+  let parameterLogStatus: ModelStatus = 'none';
+  let resourceLog: ModelLog | null = null;
+  let resourceLogStatus: ModelStatus = 'none';
+  let status: ModelStatus = 'none';
+
+  $: {
+    if (model) {
+      const {
+        refresh_activity_type_logs: activityLogs,
+        refresh_model_parameter_logs: parameterLogs,
+        refresh_resource_type_logs: resourceLogs,
+      } = model;
+
+      activityLog = activityLogs[0] ?? null;
+      parameterLog = parameterLogs[0] ?? null;
+      resourceLog = resourceLogs[0] ?? null;
+
+      if (activityLog) {
+        if (activityLog.success) {
+          activityLogStatus = 'complete';
+        } else {
+          activityLogStatus = 'error';
+        }
+      } else {
+        activityLogStatus = 'extracting';
+      }
+
+      if (parameterLog) {
+        if (parameterLog.success) {
+          parameterLogStatus = 'complete';
+        } else {
+          parameterLogStatus = 'error';
+        }
+      } else {
+        parameterLogStatus = 'extracting';
+      }
+
+      if (resourceLog) {
+        if (resourceLog.success) {
+          resourceLogStatus = 'complete';
+        } else {
+          resourceLogStatus = 'error';
+        }
+      } else {
+        resourceLogStatus = 'extracting';
+      }
+
+      if (activityLogStatus === 'error' || parameterLogStatus === 'error' || resourceLogStatus === 'error') {
+        status = 'error';
+      } else if (
+        activityLogStatus === 'complete' &&
+        parameterLogStatus === 'complete' &&
+        resourceLogStatus === 'complete'
+      ) {
+        status = 'complete';
+      } else if (
+        activityLogStatus === 'extracting' &&
+        parameterLogStatus === 'extracting' &&
+        resourceLogStatus === 'extracting'
+      ) {
+        status = 'extracting';
+      }
+    } else {
+      status = 'none';
+    }
+  }
+
+  function selectActivityLog() {
+    selectedLog = 'activity';
+    dispatch('select', activityLog);
+  }
+
+  function selectParameterLog() {
+    selectedLog = 'parameter';
+    dispatch('select', parameterLog);
+  }
+
+  function selectResourceLog() {
+    selectedLog = 'resource';
+    dispatch('select', resourceLog);
+  }
+</script>
+
+<div class="model-status-container">
```

Comment:
For some reason the last line here is 3px taller than the lines without errors
<img width="237" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/8548b627-dc29-45ea-b248-c1fd1725011a">